consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeCoordinator.java

```diff
@@ -88,7 +84,7 @@ public MergeCoordinator(
                 protocolContext,
                 protocolSchedule,
                 this.miningParameters.getMinTransactionGasPrice(),
-                this.miningParameters.getCoinbase().orElse(Address.ZERO),
+                address.or(miningParameters::getCoinbase).orElse(Address.ZERO),
```

Comment:
this was the cause of the bad block proposals on kintsugi.  Consistenly using suggestedFeeRecipient now, with fallback to miningParameters coinbase